tools/AzureRM.BootStrapper/AzureRM.Bootstrapper.psm1

```diff
@@ -625,6 +625,32 @@ function Find-PotentialConflict
   return $false
 }
 
+# Helper function to invoke install-module
+function Invoke-InstallModule
```

Comment:
Please update the unit tests for this change - the unit tests should no longer skip tests or change behavior based on powershell version